keras_nlp/models/distil_bert/distil_bert_classifier.py

```diff
@@ -210,3 +210,18 @@ def preprocessor_cls(cls):
     @classproperty
     def presets(cls):
         return copy.deepcopy(backbone_presets)
+
+    @classmethod
+    def from_preset(
+        cls,
+        preset,
+        num_classes,
```

Comment:
I think if we just follow @jbischof's work in CV here, we should not extend `from_preset` everywhere, and just rely on the fact we will get a somewhat descriptive error if `num_classes` is not provider via kwargs.

There is probably some work we could do with a try/catch block on the base `Task` class to improve the error message if we would like, but I don't think we want to redefine `from_preset` everywhere. The goal should be keeping the end task classes, as short and concise as we can!